tests/promela/resources/binary_operators.pml

```diff
@@ -0,0 +1,7 @@
+inline test()
+{
+    a = (5 + 6);
+    b = (a * 3);
+    c = (a - b);
+    d = (c / 2);
+}
```

Comment:
missing `\n` at the EOF, just creates "triggering" warning flag from GitHub while reviewing.

Reply:
ACK.
And here the algo for comparing files (instead of a zip...) comes in handy